pkgs/os-specific/linux/musl/default.nix

```diff
@@ -92,7 +92,7 @@ stdenv.mkDerivation rec {
   outputs = [ "out" "dev" ];
 
   dontDisableStatic = true;
-  separateDebugInfo = true;
+  separateDebugInfo = !(stdenv.hostPlatform.useLLVM or false);
```

Comment:
This should be set for everything - separate-debug-info relies on gcc binutils

Reply:
I believe you, but am a bit surprised by this.  Is this because of the way we do things or do LLVM bits not support split debug info? Or is "split"/"separate" an important difference? O:)

Reply:
I looked into it and it looks like there are no build ids produced:

```
$CC test.c -o test
$READELF -n test
```

Probably configurable?